ext/web/compression.rs

```diff
@@ -38,26 +38,24 @@ impl Resource for CompressionResource {
   }
 }
 
-#[op]
+#[op(fast)]
 pub fn op_compression_new(
   state: &mut OpState,
-  format: String,
+  format: u32,
   is_decoder: bool,
-) -> ResourceId {
+) -> u32 {
   let w = Vec::new();
```

Comment:
Why always the replacing of ResourceId with a plain u32?

Reply:
umm...`Result<ResourceId, _>` isn't working

Reply:
Huh. Gotta take a look at that.